statistics/sample.go

```diff
@@ -98,6 +100,7 @@ func (c *SampleCollector) collect(sc *stmtctx.StatementContext, d types.Datum) e
 		}
 	}
 	c.seenValues++
+	c.TotalSize += float64(len(d.GetBytes()) - 1)
```

Comment:
When merging the samples,  the total size has already been updated at [here](https://github.com/pingcap/tidb/pull/5974/files#diff-7761e2fd5ef728abbb9e8cb86e3cff8eR46).